# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../execwrap/flexible_config.patch
# Copyright (C) 2006 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./execwrap_config.h.orig	2006-11-23 09:38:49.000000000 -0500
+++ ./execwrap_config.h	2006-11-23 09:39:37.000000000 -0500
@@ -9,16 +9,16 @@
 
 
 /* Our parent must have this UID, or we will abort. */
-#define PARENT_UID              104
+#define PARENT_UID              @@PARENT_UID@@
 
 /* Minimum UID we can switch to. */
-#define TARGET_MIN_UID          1000
+#define TARGET_MIN_UID          @@TARGET_MIN_UID@@
 
 /* Minimum GID we can switch to. */
-#define TARGET_MIN_GID          100
+#define TARGET_MIN_GID          @@TARGET_MIN_GID@@
 
 /* Path prefix all targets must start with. */
-#define TARGET_PATH_PREFIX      "/var/www/light/"
+#define TARGET_PATH_PREFIX      "@@TARGET_PATH_PREFIX@@"
 
 /* Default UID to switch to, if none given. */
 #define DEFAULT_UID             65534
